[fix] No text to send to TTS API 오류

## 🤔 오류 내용

에러로그 함께 입력  
```
[2024-07-17 05:40:31,042: ERROR/ForkPoolWorker-8] Task utils.celery_worker.generate_audio_from_string[14d8d6ea-7e99-49ec-991e-8062ac5dbaca] raised unexpected: AssertionError('No text to send to TTS API')
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/celery/app/trace.py", line 453, in trace_task
    R = retval = fun(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/celery/app/trace.py", line 736, in __protected_call__
    return self.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/utils/celery_worker.py", line 76, in generate_audio_from_string
    tts.save(sample_path)
  File "/usr/local/lib/python3.12/site-packages/gtts/tts.py", line 335, in save
    self.write_to_fp(f)
  File "/usr/local/lib/python3.12/site-packages/gtts/tts.py", line 316, in write_to_fp
    for idx, decoded in enumerate(self.stream()):
  File "/usr/local/lib/python3.12/site-packages/gtts/tts.py", line 263, in stream
    prepared_requests = self._prepare_requests()
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/gtts/tts.py", line 210, in _prepare_requests
    assert text_parts, "No text to send to TTS API"
           ^^^^^^^^^^
AssertionError: No text to send to TTS API
```
<br>

## ⚠ 에러 캡쳐
없음

<br>

--- a/app/routers/chat.py
+++ b/app/routers/chat.py
@@ -111,8 +111,10 @@ How would you respond to the question: "{question}"?
                 db, chatroom_id=chatroom_id, is_user=False, content=server_message
             )
 
+            # 음성을 생성하는 celery task 실행
             task_audio = celery_worker.generate_audio_from_string.delay(
-                re.sub(r"[^\uAC00-\uD7A30-9a-zA-Z\s]", "", server_message)
+                # 한글, 영어, 숫자, 공백만 남기고 제거
+                re.sub(r"[^\uAC00-\uD7A3a-zA-Z0-9 ]", "", server_message)
             )
 
             server_audio = task_audio.get()

